js/basicblocks.js

```diff
@@ -3743,6 +3743,23 @@ function initBasicProtoBlocks(palettes, blocks, beginnerMode) {
         newblock.hidden = true;
     }
 
+    var newblock = new ProtoBlock('distance');
+    newblock.palette = palettes.dict['number'];
+    blocks.protoBlockDict['plus'] = newblock;
+    newblock.fontsize = 28;
+    newblock.staticLabels.push('||');
+    newblock.fourArgMathBlock();
+    newblock.dockTypes[0] = 'anyout';
```

Comment:
no need to override the default here-- it will always e numberout

Reply:
So , do I need to delete these lines?

Reply:
Just the first one.